template.yml

```diff
@@ -1,4 +1,4 @@
-Transform: AWS::Serverless-2016-10-31
+.Transform: AWS::Serverless-2016-10-31
```

Comment:
There is an extra dot here, could you remove this and then I'll be happy to merge the pull request. Thank you for identifying the bug!

Reply:
Oh weird, I definitely did not mean to add that, nice catch!

Reply:
Removed in the next commit